src/ide/ty.rs

```diff
@@ -8,9 +8,82 @@ use cairo_lang_semantic::{
     },
 };
 
-use cairo_lang_utils::{LookupIntern, ordered_hash_map::OrderedHashMap};
+use cairo_lang_syntax::node::{
+    ast::{Expr, GenericArg, GenericArgValue, TerminalUnderscore, UnaryOperator},
+    helpers::PathSegmentEx,
+};
+use cairo_lang_utils::LookupIntern;
+use cairo_lang_utils::ordered_hash_map::OrderedHashMap;
 use itertools::Itertools;
 
+// Find all `_` in provided type caluse.
+pub fn find_underscores(
```

Comment:
this is called a `wildcard`, rust ref for reference: https://doc.rust-lang.org/stable/reference/patterns.html?highlight=pattern#wildcard-pattern